GDAS-validation/run_job.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
While I understand the utility of `run_job.sh`, I personally find it much easier to `cd EXPDIR` and manually type rocoto commands.   The _hands on_ approach has the added benefit of training users how to directly control parallel job steps.   This skill is useful when they move beyond gdas validation and begin running cycled experiments